recipe/run_test.bat

```diff
@@ -0,0 +1,5 @@
+%PYTHON% -m pip check
+cd test
+:: test_audio_examples uses an uninstalled local folder ("examples");
```

Comment:
isn't it just as easy as doing `mkdir $PATH/examples` or are there files expected in that directory?

Reply:
There are files expected.